Content.Server/CartridgeLoader/Cartridges/MessagesCartridgeSystem.cs

```diff
@@ -0,0 +1,310 @@
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Shared.PDA;
+using Robust.Shared.Containers;
+using Robust.Shared.Map;
+using Content.Server.Radio.Components;
+using Content.Server.GameTicking;
+using Robust.Shared.Timing;
+using Content.Shared.Access.Components;
+using Content.Server.Radio.EntitySystems;
+using Content.Server.Power.Components;
+using Content.Shared.Chat;
+using Robust.Shared.Player;
+using Content.Server.PDA.Ringer;
+using Robust.Shared.Network;
+
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class MessagesCartridgeSystem : EntitySystem
+{
+    [Dependency] private readonly CartridgeLoaderSystem _cartridgeLoaderSystem = default!;
+    [Dependency] private readonly MessagesServerSystem _messagesServerSystem = default!;
+    [Dependency] private readonly IGameTiming _gameTiming = default!;
+    [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
```

Comment:
Just cache the system directly.

Reply:
I honestly lack the knowledge to understand what exactly you mean.

Reply:
You never need IEntitySystemManager on a system.